libraries/AP_Baro/AP_Baro_MS5611.cpp

```diff
@@ -264,9 +264,10 @@ void AP_Baro_MS56XX::_timer(void)
     }
 
     /* if we had a failed read we are all done */
-    if (adc_val == 0) {
+    if (adc_val == 0 || adc_val == 0xFFFFFF) {
```

Comment:
`next_state` is still updated in this new case you're handling.  Presumably that will recover in time.

Reply:
we don't actually know the sensor state, so we can't tell if update is right or not